pkg/skaffold/deploy/kubectl.go

```diff
@@ -64,6 +64,8 @@ func (k *KubectlDeployer) Labels() map[string]string {
 // Deploy templates the provided manifests with a simple `find and replace` and
 // runs `kubectl apply` on those manifests
 func (k *KubectlDeployer) Deploy(ctx context.Context, out io.Writer, builds []build.Artifact) ([]Artifact, error) {
+	logrus.Debugln("kubectl client version:", k.kubectl.Version())
```

Comment:
nit: I would even consider making this `Info` level log

Reply:
Good point